nf_core/bump_version.py

```diff
@@ -44,10 +44,10 @@ def bump_pipeline_version(lint_obj, new_version):
     nfconfig_newstr = "container = 'nfcore/{}:{}'".format(lint_obj.pipeline_name.lower(), docker_tag)
     update_file_version("nextflow.config", lint_obj, nfconfig_pattern, nfconfig_newstr)
 
-    # Update travis image tag
-    nfconfig_pattern = r"docker tag nfcore/{name}:dev nfcore/{name}:(?:{tag}|dev)".format(name=lint_obj.pipeline_name.lower(), tag=current_version.replace('.',r'\.'))
-    nfconfig_newstr = "docker tag nfcore/{name}:dev nfcore/{name}:{tag}".format(name=lint_obj.pipeline_name.lower(), tag=docker_tag)
-    update_file_version(".travis.yml", lint_obj, nfconfig_pattern, nfconfig_newstr)
+    # Update GitHub Actions CI image tag
+    nfconfig_pattern = r"docker pull nfcore/{name}:dev && docker tag nfcore/{name}:dev nfcore/{name}:(?:{tag}|dev)".format(name=lint_obj.pipeline_name.lower(), tag=current_version.replace('.',r'\.'))
```

Comment:
Why do you need this extra bit? The regex should work on partial strings, so I don't think it's necessary to include the entire line...

Plus I want to make it in to two lines instead of chaining with a `&&` soon as it's tidier, and I don't want this code to break when I do 😆 

```suggestion
    nfconfig_pattern = r"docker tag nfcore/{name}:dev nfcore/{name}:(?:{tag}|dev)".format(name=lint_obj.pipeline_name.lower(), tag=current_version.replace('.',r'\.'))
```

Reply:
I did play with changing that in `ci.yml` but I got annoyed with `bump-version` not working and added it back in. Then I just forgot what I was doing altogether 💥

Reply:
See https://github.com/nf-core/tools/pull/502/commits/5d21c40a0c9b0b1519f7035ada7a346925aaae7b